### PR TITLE
Bring Development to master

### PR DESCRIPTION
**Pull request by agusaldasoro**
- Expose Auth Server
- Implement Rate Limit using Proof of Work as JWT

### CODE REVIEW

**Comment by menduz** [comment]
@agusaldasoro Do deployed catalyst already support the PoW authentication client-side? Checking to prevent rate limiting content synchronization

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -11,6 +11,9 @@ jobs:
           # - https://localhost.decentraland.services
     steps:
       - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '12'
```
Comment (jmoguilevsky) [inline .github/workflows/ci.yml:16]: why 12 and not 14?

**Comment by agusaldasoro** [comment]
> @agusaldasoro Do deployed catalyst already support the PoW authentication client-side? Checking to prevent rate limiting content synchronization

In this change the rate limit is configured as `dry run` so this change won't limit any traffic. Anyway, we will roll out this change among the change in catalysts to support Pow if configured.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (agusaldasoro) [reply .github/workflows/ci.yml:16]: I needed 12 for the solve challenge to work

**Review by nchamo — APPROVED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (agusaldasoro) [reply .github/workflows/ci.yml]: Fixed!

**Review by jmoguilevsky — APPROVED** [review]

**Pull request merged**